### PR TITLE
DOCUMENTATION: Finished Variables section.

### PR DESCRIPTION
**Pull request by muhammad-abdulloh**
DOCUMENTATION: Finished Variables page continue keep learning 🥇

### CODE REVIEW

File: Variables.md [hunk 1]
```diff
@@ -101,3 +101,68 @@ Student student = GetStudent();
...
+{
+    Name = "Muhammadabdulloh",
+    Score = 100
+};
+```
+<br /> <br />
+
+### 0.2 Tashkillashtirish
+
+#### 0.2.0 Sindirish
```
Comment (ElbekDeveloper) [inline Variables.md]: Keyingi qatorga o'tkazish

File: Variables.md [hunk 2]
```diff
@@ -101,3 +101,68 @@ Student student = GetStudent();
 var student = GetStudent();
 ```
 <br />
+
+
+#### 0.1.2 Toza bo'lmagan tiplar
```
Comment (ElbekDeveloper) [inline Variables.md]: Aniq bo'lmagan tiplar

**Review by ElbekDeveloper — CHANGES_REQUESTED** [review]

File: Variables.md (hunk 2 above)
Comment (muhammad-abdulloh) [reply Variables.md]: aha

Comment (muhammad-abdulloh) [reply Variables.md]: O'zgartirishlar kiritib qo'ydim

**Comment by ElbekDeveloper** [comment]
`DOCUMENTATIONS: Resolve Comments` chiroyliroq commit

**Review by ElbekDeveloper — APPROVED** [review]

**Pull request merged**